plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2954,6 +2954,17 @@ public void testSplitPruningForFilterOnNonPartitionColumn(DataMappingTestSetup t
         }
     }
 
+    @Test
+    public void testGetTableProperties()
+    {
+        assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT) WITH (format='PARQUET')");
```

Comment:
Add `partitioning` to show the `$properties` is **not**  the contents of the `WITH (...)` clause.